packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VXTooltip/VXTooltip.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withTheme } from 'styled-components'
+import { Tooltip } from '@vx/tooltip'
+import { withBoundingRects, withBoundingRectsProps } from '@vx/bounds'
+import { Box } from 'grommet'
+import { SpacedText } from '@zooniverse/react-components'
+import Triangle from '../../../../shared/Triangle'
+
+function getNumberValue (string) {
+  const splitString = string.split('px')
```

Comment:
This will break if fonts are defined in ems, rems or points.

Reply:
Yeah, it's not ideal. It's assuming that we're using the zoo theme, which is using px.